tracer/src/Datadog.Trace.Tools.Shared/System.Diagnostics.CodeAnalysis.Attributes.cs

```diff
@@ -0,0 +1,172 @@
+// <copyright file="System.Diagnostics.CodeAnalysis.Attributes.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// https://github.com/dotnet/runtime/blob/bffa7cf52b3982597adc5447c25e7aaa3b063c1c/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs
+
+// This file contains attributes from the System.Diagnostics.CodeAnalysis namespace
+// used by the compiler for null-state static analysis.
+// This is a C# feature, but requires these attributes to be defined,
+// so we define them here for older .NET runtimes.
+// https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/nullable-analysis
+
+#pragma warning disable SA1649 // file name should match first type name
```

Comment:
Can we just link this file in from the Datadog.Trace version? Doesn't seem worth duplicating all these again?

Reply:
Good idea